020-LinuxFundamentals/README.md

```diff
@@ -0,0 +1,95 @@
+# What The Hack - Linux Fundamentals
+
+## Introduction
+This hackathon will serve as an introduction to important command line concepts and skills on Linux environments.
+
+<img align="right" src="./Student/resources/images/linuxpenguin.png" width="250"/>
+
+## Linux History
+
+Linux is a family of free and open-source operating systems based on the Linux kernel. Operating systems based on Linux are known as Linux distributions or distros. Examples include Debian, Ubuntu, Fedora, CentOS, Gentoo, Arch Linux, and many others.
+
+The Linux kernel has been under active development since 1991 and has proven to be extremely versatile and adaptable. You can find computers that run Linux in a wide variety of contexts all over the world, from web servers to cell phones. Today, 90% of all cloud infrastructure and 74% of the world’s smartphones are powered by Linux.
+
+To read more about Linux History, Linux Distributions and Linux Kernel, [click here](./Student/resources/linux-history.md).
+
+
+## Learning Objectives
+In this hack you will be challenged with some common tasks from a real world scenario in Linux administration duties, such as:
+
+1. Create a Linux Virtual Machine on Azure
+2. Handle files and directories
+3. Maninpulate file contents
+4. Work with standard Linux permissions
+5. Collect information about Linux processes in your environment
+6. Management of users and groups
+7. Basic shell scripting 
+8. Work with disks, partitions and logical volume manager
+9. Linux package management 
+10. Implement a basic webserver 
+
+## Challenges
+
+With the exception of challenge 01 which has as an outcome a Linux Virtual Machine which will be required for all other challenges, each challenge can be done separately and they are not interdependent. The level of complexity increases accordingly with the respective number of each one.
+
+1. Challenge 01: **[Create a Linux Virtual Machine](Student/Challenge-01.md)**
```

Comment:
The table of contents should be a bullet list, not an ordered list.  (The challenge #s are already there in the text.  This was an error in the V1 template generator that we updated already.  Thanks for testing!)